src/net.cpp

```diff
@@ -1216,7 +1216,7 @@ void CConnman::NotifyNumConnectionsChanged()
     }
 }
 
-void CConnman::InactivityCheck(CNode *pnode)
+void CConnman::InactivityCheck(CNode *pnode) const
```

Comment:
1c65c075 this change to const member function for `net.{h,cpp}::CConnman::InactivityCheck`, in which `pnode->fDisconnect` can be mutated, I find a tiny bit odd in terms of displayed intent

Reply:
Given `CConnman c` do we agree that the logical state of `c` is unaffected by `c.InactivityCheck(p)`? :)

That `p` can be modified by a call `c.InactivityCheck(p)` shouldn't be odd since that follows from the signature, no? :)